packages/http-client-csharp/generator/Microsoft.Generator.CSharp.ClientModel/test/Mocks/MockHelpers.cs

```diff
@@ -57,18 +59,23 @@ internal static class MockHelpers
                 mockTypeFactory.Protected().Setup<CSharpType>("CreateCSharpTypeCore", ItExpr.IsAny<InputType>()).Returns(createCSharpTypeCore);
             }
 
-            var configFilePath = Path.Combine(AppContext.BaseDirectory, MocksFolder);
             // initialize the mock singleton instance of the plugin
             var codeModelInstance = typeof(CodeModelPlugin).GetField("_instance", BindingFlags.Static | BindingFlags.NonPublic);
             var clientModelInstance = typeof(ClientModelPlugin).GetField("_instance", BindingFlags.Static | BindingFlags.NonPublic);
             // invoke the load method with the config file path
             var loadMethod = typeof(Configuration).GetMethod("Load", BindingFlags.Static | BindingFlags.NonPublic);
-            object?[] parameters = [configFilePath, null];
+            object?[] parameters = [ConfigFilePath, null];
             var config = loadMethod?.Invoke(null, parameters);
             var mockGeneratorContext = new Mock<GeneratorContext>(config!);
             var mockPluginInstance = new Mock<ClientModelPlugin>(mockGeneratorContext.Object) { CallBase = true };
             mockPluginInstance.SetupGet(p => p.TypeFactory).Returns(mockTypeFactory.Object);
 
+            if (inputLibrary is not null)
+            {
+                var inputLibInstance = typeof(CodeModelPlugin).GetField("_inputLibrary", BindingFlags.Instance | BindingFlags.NonPublic);
```

Comment:
Should we make InputLibrary virtual so that it can be mocked like the other properties?

Reply:
sure